tests/experimental/datasources/test_integration_datasource.py

```diff
@@ -515,3 +518,26 @@ def test_batch_head(
         assert n_rows_validation_error in str(
             e.value
         ) or fetch_all_validation_error in str(e.value)
+
+
+def test_sql_query_data_asset(empty_data_context):
+    context = empty_data_context
+    datasource = sqlite_datasource(context, "yellow_tripdata.db")
+    passenger_count_value = 5
+    asset = (
+        datasource.add_query_asset(
+            name="query_asset",
+            query=f"SELECT * from yellow_tripdata_sample_2019_02 WHERE passenger_count = {passenger_count_value}",
```

Comment:
@billdirks Per earlier comments, I would want to be able to include a free amount of blank spaces in the `SQL` query specification.

Reply:
Added prepended spaces.